server-common/src/main/java/org/apache/kafka/server/config/ServerTopicConfigSynonyms.java

```diff
@@ -29,7 +29,7 @@
 import org.apache.kafka.common.utils.Utils;
 
 public final class ServerTopicConfigSynonyms {
-    private static final String LOG_PREFIX = "log.";
+    public static final String LOG_PREFIX = "log.";
     private static final String LOG_CLEANER_PREFIX = LOG_PREFIX + "cleaner.";
```

Comment:
Is `LOG_CLEANER_PREFIX` more suitable to be the prefix for `CleanerConfig`?

Reply:
`CleanerConfig` are in `storage` module which is not accessbile from `server-common` were `ServerTopicConfigSynonyms`

Reply:
Sorry that I meant `CleanerConfig` can use `LOG_CLEANER_PREFIX` instead of `LOG_PREFIX`. For example:
```java
ServerTopicConfigSynonyms.LOG_PREFIX + "cleaner.threads";
```
could be replaced by 
```java
ServerTopicConfigSynonyms.LOG_CLEANER_PREFIX + "threads";
```
It seems to me the later is more readable. Or `CleanerConfig` can have a new constant "LOG_CLEANER_PREFIX"

Reply:
Great catch. Thanks. I replaced constant with more specific one.